torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -571,17 +572,20 @@ def _annotate_maxpool2d(
             input_act = maxpool_node.args[0]  # type: ignore[union-attr]
             assert isinstance(input_act, Node)
 
-            input_act_qspec = get_input_act_qspec(quantization_config)
-            output_act_qspec = get_output_act_qspec(quantization_config)
+            # only annotate maxpool when the input node is annotated
+            if "quantization_annotation" not in input_act.meta or not input_act.meta["quantization_annotation"]._annotated:
```

Comment:
To to more specific, I think we should check the `input_act.meta["quantization_annotation"].output_qspec is not None`.  Since we may only annotate the input but not output of the `input_act`.

Reply:
oh OK makes sense, thanks!